py3status/modules/deadbeef.py

```diff
@@ -3,95 +3,101 @@
 Display song currently playing in deadbeef.
 
 Configuration parameters:
-    cache_timeout: how often we refresh usage in seconds (default 1)
-    delimiter: delimiter character for parsing (default '¥')
-    format: see placeholders below (default '{artist} - {title}')
+    cache_timeout: refresh interval for this module (default 1)
+    delimiter: delimiter between metadata fields (default '¥å¥å¥å')
+    format: display format for this module (default '{artist} - {title}')
 
 Format placeholders:
-    {artist} artist
-    {title} title
-    {elapsed} elapsed time
-    {length} total length
-    {year} year
-    {tracknum} track number
+    {artist}        artist
+    {length}        length
+    {playback_time} playback time
+    {title}         title
+    {tracknumber}   track number
+    {year}          year
```

Comment:
Remove the descriptions? For instance, `track number` is not same as `tracknumber` so I should rename it to `tracknumber`. Same thing for `playback_time`. A small change like this could mean `5` or `05` for tracknumber.